06-variables/01-concat/.README.md

```diff
@@ -0,0 +1,3 @@
+# concat variables
+
+Write a small function that accepts two arguments ({{index . "var1"}}, {{index . "var2"}}) and returns their concatenated forms where the first returned value is {{index . "var1"}}{{index . "var2"}} and the second value is {{index . "var2"}}{{index . "var1"}}. Place your code in the `exercise.go` file near tho the placeholder `// INSERT YOUR CODE HERE`
```

Comment:
The same problem as in the exercise of Tamas: you ask students to write the *same* program and then you create *per-user* tests. This does not make too much sense. We want them to write one of 2-3 different programs and then we should prepare specific tests for each possible program. 

Here is one way to do that: let the exercise be to write a greeter service that asks for a name and then returns "Hi <name>" or similar, where "Hi" may be in one of 3-4 languages. We pick one language randomly for each student and then we prepare tests that will ask for the returned value to match "Hi X" for X in the chose language.